lib/src/view_models/connected_bluetooth_device_screen_view_model.dart

```diff
@@ -38,15 +40,14 @@ class ConnectedBluetoothDeviceScreenViewModel extends ChangeNotifier {
     required this.tipsDialogCtaText,
   }) : _connectBluetoothDeviceRepository = connectBluetoothDeviceRepository;
 
-  Future<void> readNetworkList(BuildContext context) async {
+  Future<void> readNetworkList(BuildContext? context) async {
     try {
       isLoadingNetworks = true;
-      wifiNetworks.clear();
-      await Future.delayed(const Duration(milliseconds: 500)); // delay to see "scanning" ui
```

Comment:
got rid of this, really not needed, you can still see loading and it's better to show these as fast as we can